webapp/channels/src/components/pricing_modal/content.tsx

```diff
@@ -220,10 +217,10 @@ function Content(props: ContentProps) {
                                     renderLastDaysOnTrial={true}
                                 />) : undefined}
                         buttonDetails={enterpriseBtnDetails()}
-                        planTrialDisclaimer={(!isPostTrial && isAdmin && !cloudFreeDeprecated) ? <StartTrialCaution/> : undefined}
-                        contactSalesCTA={(isPostTrial || !isAdmin || cloudFreeDeprecated) ? undefined : <ContactSalesCTA/>}
+                        planTrialDisclaimer={undefined}
+                        contactSalesCTA={(isPostTrial || !isAdmin) ? undefined : <ContactSalesCTA/>}
```

Comment:
Same thing on this line. The surrounding lines look correct though

Reply:
Same thing here - pre July 2023 we wanted this CTA to show for those on Cloud Free to get them to move to enterprise. Post July 2023, they can't be on Cloud Free so we're removing that (which would have always been true) and leaving the logic in place for how it's always been for self hosted.